src/screens/faq/styles.tsx

```diff
@@ -0,0 +1,100 @@
+import { makeStyles } from '@material-ui/styles';
+import { useTheme } from '@material-ui/core/styles';
+import { getMinMediaQuery } from '@styles';
+
+export const useGetStyles = () => {
+  const theme: any = useTheme();
+  const useStyles = makeStyles({
+    root: {
+      padding: '1rem 1rem 2.5rem',
+    },
+    title: {
+      fontSize: '1.375rem',
+      fontWeight: 500,
+      margin: '0',
+      marginBottom: '0.5rem',
+      marginTop: '1rem',
+      lineHeight: '1.375rem',
+      color: '#000000',
+      [getMinMediaQuery(theme.breakpoints.tablet)]: {
+        fontSize: '1.5rem',
+        marginBottom: '1rem',
+        marginTop: '1.5rem',
+      },
+      [getMinMediaQuery(theme.breakpoints.desktop)]: {
+        fontSize: '1.75rem',
+        marginBottom: '1.125rem',
+        marginTop: '2.5rem',
+      },
+    },
+    context: {
+      fontSize: '0.875rem',
+      margin: '0',
+      lineHeight: '1.375rem',
```

Comment:
Can you remove the line heights? we can set those in the theme later. Would be a hassle to suddenly decide we want some other lineheight

Reply:
done

Reply:
done